bears/js/PrettierLintBear.py

```diff
@@ -23,13 +23,17 @@ class PrettierLintBear:
                   'Multi-line objects'}
     SEE_MORE = 'https://prettier.io/'
 
-    regex = re.compile(r'L(?P<line>\d+)C(?P<column>\d+): (?P<message>.*)')
+    regex = re.compile(
+        r'(?P<message>SyntaxError:.*) '
+        r'\((?P<line>\d+):(?P<column>\d+)\)')
 
     def process_output(self, output, filename, file):
         stdout, stderr = output
-        yield from self.process_output_corrected(stdout, filename, file)
-        yield from self.process_output_regex(stderr, filename, file,
-                                             self.regex)
+        if stdout == '\n':
```

Comment:
why is this `if` needed?

Reply:
When there is a syntax error stdout has a `\n`. So I suppose we should only show the syntax error in this case.

Reply:
But the regex being applied to `stdout` will ignore the `\n`..?

Reply:
no we are using `process_output_corrected` since the bear prints corrected output to `stdout`

Reply:
That feels like a bug in `process_output_corrected`.  If there was an error, the linter can only emit `\n` on stdout.  It shouldnt decide that means the file should be blanked.

Reply:
fwiw https://github.com/coala/coala/issues/5415